import-service/src/functions/importProductsFile/index.ts

```diff
@@ -15,6 +15,13 @@ export default {
             },
           },
         },
+        authorizer: {
+          name: "basicAuthorizer",
+          arn: "arn:aws:lambda:eu-west-1:662624365868:function:authorization-service-dev-basicAuthorizer",
```

Comment:
guess that `arn`s should be in env file and be hidden. Am I wrong?

Reply:
Actually, you could use `${aws:region}` and `${aws:accountId}` shortcuts instead of hardocdding it. It's kinda sensitive info, that is best kept hidden I guess.